src/openrct2/object/PeepNamesObject.cpp

```diff
@@ -0,0 +1,48 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2024 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#include "PeepNamesObject.h"
+
+#include "../Context.h"
+#include "../PlatformEnvironment.h"
+#include "../core/Json.hpp"
+
+using namespace OpenRCT2;
+
+void PeepNamesObject::Load()
+{
+}
+
+void PeepNamesObject::Unload()
+{
+}
+
+void PeepNamesObject::ReadJson(IReadObjectContext* context, json_t& root)
+{
+    Guard::Assert(root.is_object(), "PeepNamesObject::ReadJson expects parameter root to be an object");
```

Comment:
Our Guards are compiled at all times, as far as I know.

Reply:
They are, in this case we would see the dialog that something failed, but if the user chooses to ignore, it would then immediately crash, would it not?

Reply:
I'm not sure it's possible to ignore? Either way, I'm just following the same practice used in e.g. FootpathRailingsObject, FootpathSurfaceObject, so I'd prefer not to make it more complicated than this.